docs/concepts/rfai.md

```diff
@@ -32,61 +32,65 @@ page_nav:
 
 ---
 
-The Request for AI portal is a platform that allows any member of the community to make requests for AI services to be published on the SingularityNET Registry.
-These requests are backed by AGI tokens, which are transferred to the developer who submits a valid solution for the request.
+The Request for AI Portal (RFAI) lets users make requests for AI services that they would like to see built and deployed onto the SingularityNET Network. Users can back requests with AGI tokens as a reward for having a service developed or as a pledge to make use of the service once it goes live. The AGI tokens are transferred to the developer who submits a valid solution for the request. 
+The RFAI portal fosters the community by enabling users to incentivize developers to publish services.
 
-## Request for AI Service
 
-* Anyone should be able to request for a service in the Web Portal;
-* Requester can back on the Service Request;
-* Service Request details will be available in IPFS and the hash will be in BlockChain Contract;
-* There is an expiry for the Service Request;
-* When a new requested is created it will be in a Open State until foundation is Approved;
-* Requester can extend the request only when it is not approved means in Open state.
+## Requesting for an AI Service
 
-## Request Approval
+* Any request for an AI service should be backed by AGI tokens.
+* Requests should be detailed enough to allow discussion and development and should be in the form of a github pull requests to repo. You can view the template for the request here
+* Provide an expiry date for the request. Meaning the date post which you can withdraw your funds if no submission has been made
+* A request created will need to be approved by the foundation before it shows up for everybody
+
+## Backing an AI Service
+* In order to request for an AI service, you need to back it with AGI tokens
+* You first need to transfer tokens to the RFAI escrow from the Account tab. This is a two step process
+    * Approve the number of tokens to deposit
+    * Deposit the tokens
+* You are now set to request for an AI service or back any approved request
```

Comment:
"... set to *make* a request for an AI... "

Reply:
Done